pkg/sfu/streamtrackermanager.go

```diff
@@ -119,9 +148,11 @@ func (s *StreamTrackerManager) AddTracker(layer int32) *StreamTracker {
 	})
 
 	s.lock.Lock()
+	paused := s.paused
 	s.trackers[layer] = tracker
 	s.lock.Unlock()
 
+	tracker.SetPaused(paused)
```

Comment:
`paused` also caused trouble with stream tracker. It was not passed in when a tracker was created. With FF, if the camera is disabled soon after start, layer 2 (the 1 FPS layer) starts up after pausing. That triggers `OnTrack` and a tracker is created. But, that tracker did not get the `paused` setting and proceeded to declare layer available seeing packets/frames from that layer.